influxdb3_server/src/http.rs

```diff
@@ -752,6 +814,17 @@ pub(crate) struct WriteParams {
     pub(crate) precision: Precision,
 }
 
+impl From<iox_http::write::WriteParams> for WriteParams {
+    fn from(legacy: iox_http::write::WriteParams) -> Self {
+        Self {
+            db: legacy.namespace.to_string(),
+            // legacy behaviour was to not accept partial:
+            accept_partial: false,
```

Comment:
This won't affect the v3 endpoint correct? That should be set to true automatically

Reply:
v2 accepted partial writes and would send any invalid writes into a system bucket called `_monitoring`. We don't want to replicate the monitoring bucket thing, but we should probably make it accept partial writes. Insanely, the v2 write API returns a 204 even if some of the writes were invalid. It only returned a 400 if everything was invalid (at least I think that was the behavior).

I don't believe v1 accepted partial writes, but I honestly don't remember at this point. Will probably require digging through docs and/or code.

Or the easiest thing might just be to test both of these out. We'll want to replicate whatever the open source builds are doing. I know that v2 open source and v2 cloud weren't quite consistent (again 🤷 )

Reply:
I created this issue to track: https://github.com/influxdata/influxdb/issues/24850